### ISSUE TITLE
Password generation

### ISSUE THREAD

**Issue by hartois** [YiannisBourkelis/Andama-Remote-Desktop#35]
On Windows Andama on every execution, on different systems generate different ID, but same password.

**Comment by hartois** [comment]
Who generate password Proxy or client?
Help me please find place in sources

**Comment by YiannisBourkelis** [comment]
@hartois strange, password is generated on the client side, each time the application runs.
Probably it has to do with the internal implementation of random numbers distribution, inside the mingw.
Tested with compilers on osx/clang, linux/g++ and windows/vs c++ and works fine.

To find the problem with the mingw compiler, I think you should look inside the function where the random password is generated. It is located in 2 places: 
1. clientserver::generateRandomPassword(int length)
2. std::string clientServerProtocol::generateRandomPassword(int length)

Probably with Mingw it always picks the same "Random" characters

**Comment by hartois** [comment]
Yes, after searching I found that problem in using rd() function in Mingw compiller.
For me I replace it to time(NULL) and get different passwords...
```
std::mt19937 mt = std::mt19937(time(NULL));
```
Сan you suggest some kind of another, more correct way?

Thanks

**Comment by YiannisBourkelis** [comment]
@hartois I don't know if this way it produces completely random passwords. Maybe std::mt19937 mt = std::mt19937(time_t(NULL)); might be better to clean the warning. It still needs investigation if it is ok.
